config/locales/hyrax.es.yml

```diff
@@ -0,0 +1,8 @@
+es:
+  hyrax:
+    product_name: 'Hydra-en-una-Caja'
+    product_twitter_handle: "@HydraInABox"
+    institution_name:   'Hydra en una Caja'
+    account_name:       "Hydra-en-una-Caja Cuenta ID"
+    directory:
+      suffix:           "@hydrainabox.org"
```

Comment:
Not inclined to take this on as part of this branch, since it's unrelated to making Hyku more `es`-friendly.